### ISSUE TITLE
Fatal Exception: NSInvalidArgumentException

### ISSUE THREAD

**Issue by andrew-andilevko-itechart** [wmcmahan/react-native-calendar-events#258]
```
Fatal Exception: NSInvalidArgumentException
0  CoreFoundation                 0x1b62a927c __exceptionPreprocess
1  libobjc.A.dylib                0x1b54839f8 objc_exception_throw
2  CoreFoundation                 0x1b6222ce8 _CFArgv
3  CoreFoundation                 0x1b61aa9a8 -[__NSPlaceholderDictionary initWithObjects:forKeys:count:]
4  CoreFoundation                 0x1b619b584 +[NSDictionary dictionaryWithObjects:forKeys:count:]
5  GlossGeniusApp                 0x104c833b4 -[RNCalendarEvents serializeCalendarEvent:]
6  GlossGeniusApp                 0x104c81638 -[RNCalendarEvents serializeCalendarEvents:]
7  GlossGeniusApp                 0x104c84b00 __71-[RNCalendarEvents fetchAllEvents:endDate:calendars:resolver:rejecter:]_block_invoke
8  libdispatch.dylib              0x1b5ce8a38 _dispatch_call_block_and_release
9  libdispatch.dylib              0x1b5ce97d4 _dispatch_client_callout
10 libdispatch.dylib              0x1b5cc4dec _dispatch_lane_serial_drain$VARIANT$armv81
11 libdispatch.dylib              0x1b5cc592c _dispatch_lane_invoke$VARIANT$armv81
12 libdispatch.dylib              0x1b5ccde08 _dispatch_workloop_worker_thread
13 libsystem_pthread.dylib        0x1b5ec9114 _pthread_wqthread
14 libsystem_pthread.dylib        0x1b5ecbcd4 start_wqthread
```

### Environment
react: 16.6.3
react-native: 0.59.8
react-native-calendar-events: 1.7.3

### Device
iPhone X
iOS 12.3.1

**Comment by karimbutt** [comment]
+1 I'm having this issue too

**Comment by gcrabtree** [comment]
This is causing issues for us as well. It's proving to be very difficult for us to debug.

**Comment by karimbutt** [comment]
@gcrabtree same for us - hard to debug. Any luck on this?

**Comment by mtimmens** [comment]
@karimbutt the commit referenced in this pull request #253 resolved the issue for us. We are currently building our app off that branch, but we are looking forward to seeing this merged and into a release.

Specify the commit directly in your package.json to test:
```
"react-native-calendar-events": "HoneyBook/react-native-calendar-events#e1e0aef84d20a3b04c089b733a5329b5a0f676e0",
```

**Comment by zach-luman** [comment]
any update on this bug?

**Comment by robertherber** [comment]
Please merge this pull request. Spent a lot of time tracking down this bug.

**Comment by daviaugustos** [comment]
Any updates about this?

**Comment by RajeshRam** [comment]
Did we have any update on these ?

**Comment by alindao-charles** [comment]
Any updates about this? I'm having this issue also.

**Comment by Zishrodrigues** [comment]
> @karimbutt the commit referenced in this pull request #253 resolved the issue for us. We are currently building our app off that branch, but we are looking forward to seeing this merged and into a release.
> 
> Specify the commit directly in your package.json to test:
> 
> ```
> "react-native-calendar-events": "HoneyBook/react-native-calendar-events#e1e0aef84d20a3b04c089b733a5329b5a0f676e0",
> ```

Been experiencing the same issue and the above mentioned merge fixed it for me. I had to use the Github repo as source rather than the NPM package as this merge hasn't been made into a NPM release yet. Might be worth doing a new release version to NPM 👍

**Comment by MoOx** [comment]
Closed by #253
